tests/test_backend.py

```diff
@@ -171,7 +173,10 @@ def _conv_test(self, x_val, w, strides=None, padding="VALID", dilations=None, rt
         x = tf.placeholder(tf.float32, shape=x_val.shape, name=_TFINPUT)
         conv = tf.nn.conv2d(x, kernel, strides=strides, padding=padding, dilations=dilations)
         _ = tf.identity(conv, name=_TFOUTPUT)
-        self._run_test_case([_OUTPUT], {_INPUT: x_val}, rtol=rtol)
+        try:
+            self._run_test_case([_OUTPUT], {_INPUT: x_val}, rtol=rtol)
```

Comment:
we lose the original exception info ?

Reply:
maybe we could use python function "traceback.print_exc()" to print the exception info.

while after discussion with guoliang, decided to use skipif to skip the test when tensorflow didn't implement the function.

Reply:
Its hard to get rid of the skipif() because sometimes results are just wrong.